src/app/containers/ConsentBanner/index.jsx

```diff
@@ -1,10 +1,41 @@
-import React, { useContext } from 'react';
+import React, { useContext, useEffect } from 'react';
 import { RequestContext } from '#contexts/RequestContext';
 import Canonical from './index.canonical';
 import Amp from './index.amp';
+import { EventContext } from '#contexts/EventContext';
+import { buildATIClickParams } from '../ATIAnalytics/params';
+import { sendEventBeacon, sendViewBeacon } from '../ATIAnalytics/beacon';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+const ConsentBanner = props => {
+  const requestContext = useContext(RequestContext);
+  const serviceContext = useContext(ServiceContext);
+  const eventContext = useContext(EventContext);
+
+  const params = buildATIClickParams(props, requestContext, serviceContext);
+
+  useEffect(() => {
```

Comment:
We chatted on slack about not integrating this in this PR? We can undo this stuff